examples/sat/bench_schoening.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+
+"""
+Benchmarking for variants of Schoening's algorithm.
+"""
+import logging
+from dataclasses import asdict
+from pathlib import Path
+import click
+import numpy as np
+import pandas as pd
+from bench_hillclimber import setup_default_logger
+from sat import SatInstance
+from bruteforce import bruteforce_solve
+from qubrabench.benchmark import track_queries, named_oracle
+from schoening import (
+    schoening_solve,
+    schoening_bruteforce_steps,
+)
+
+from qubrabench.utils.plotting import PlottingStrategy
```

Comment:
Like so?

```suggestion
from qubrabench.benchmark import track_queries, named_oracle
from qubrabench.utils.plotting import PlottingStrategy

from schoening import (
    schoening_solve,
    schoening_bruteforce_steps,
)
from bench_hillclimber import setup_default_logger
from sat import SatInstance
from bruteforce import bruteforce_solve
```

Reply:
yep!

Reply:
I've added `isort` which will do this automatically. Once you rebase and run make, it should get fixed.